packages/sdk-schema-generator/package.json

```
@@ -0,0 +1,42 @@
+{
+	"name": "@directus/sdk-schema-generator",
+	"version": "1.0.0",
```

Will let changeset handle the initial version. If you want to start with "1.0.0" from the beginning you can add a "major" changeset.

```suggestion
	"version": "0.0.0",
```